src/Tools/dotnet-monitor/Egress/FileSystem/FileSystemEgressProvider.cs

```diff
@@ -75,8 +78,9 @@ public FileSystemEgressProvider(ILogger<FileSystemEgressProvider> logger)
                             File.Delete(intermediateFilePath);
                         }
                     }
-                    catch (Exception)
+                    catch (Exception ex)
                     {
+                        _logger.LogTrace(ex.Message);
```

Comment:
Debug level?

Reply:
Also, add the LoggerExtensions instead of calling ILogger directly. And pass the exception through the exception parameter.